source/web_integration/pnapi_web_integration.rst

```diff
@@ -95,11 +128,11 @@ Client Side Parameters
       Set of JavaScript parameters to be captured where the key name should be RingPool parameter name.
       These values are updated every subsequent visit.
 
-  * - savePoolParams
+  * - updatePoolParams
     - Bool
-    - Default: `false`
+    - Default: `true`
 
-      When true, the integration code will not update `poolParams` on subsequent page visits.
+      When `false`, the integration code will not update `poolParams` on subsequent page visits.
```

Comment:
This documentation applies to all of `2.0.0` and above, and any changes we make are backwards compatible.. That said, we did indeed flip the contract here, but actually have  compatibility for the old parameter name. Going forward, we would like customers to use this naming convention instead.